docs/concepts/workloads/pods/pod-overview.md

```diff
@@ -55,7 +55,7 @@ You'll rarely create individual Pods directly in Kubernetes--even singleton Pods
 
 > Note: Restarting a container in a Pod should not be confused with restarting the Pod. The Pod itself does not run, but is an environment the containers run in and persists until it is deleted.
 
-Pods do not, by themselves, self-heal. If a Pod is scheduled to a Node that fails, or if the scheduling operation itself fails, the Pod is deleted; likewise, a Pod won't survive an eviction due to a lack of resources or Node maintenance. Kubernetes uses a higher-level abstraction, called a *Controller*, that handles the work of managing the relatively disposable Pod instances. Thus, while it is possible to use Pod directly, it's far more common in Kubernetes to manage your pods using a Controller. See [Pods and Controllers](#pods-and-controllers) for more information on how Kubernetes uses Controllers to implement Pod scaling and healing.
+Pods do not, by themselves, self-heal. If a Pod is scheduled to a Node that fails, or if the scheduling operation itself fails, the Pod is deleted; likewise, a Pod won't survive an eviction due to a lack of resources or Node maintenance. Kubernetes uses a higher-level abstraction, called a *Controller*, that handles the work of managing the relatively disposable Pod instances. Thus, while it is possible to use pods directly, it's far more common in Kubernetes to manage your pods using a Controller. See [Pods and Controllers](#pods-and-controllers) for more information on how Kubernetes uses Controllers to implement Pod scaling and healing.
```

Comment:
the "Pod" in "use Pod directly" refers to the Pod resource type (see: ).
the "pods" in "to manage your pods using a Controller" are about the Pod objects instantiated from a Pod.

So I don't think this change is necessary.

Reply:
I see. It doesn't sound very natural when read aloud as it is currently written. Would "use the Pod resource type" or "use Pods" work? There's also a usage of "pods" a few words later in "manager your pods using a Controller". Seems like by and large it's desired to use capitalized terms for resource types. I'd just like to be able to read the documentation aloud in my head without what sounds like a grammatical error.

Reply:
ya. Changing it to 'Pod resource type' sounds a better idea.